01-master.yaml

```diff
@@ -52,13 +37,14 @@ write_files:
           --anonymous-auth=false \
           --client-ca-file=/etc/kubernetes/ssl/ca.pem \
           --api-servers=http://127.0.0.1:8080 \
+          --network-plugin-dir=/etc/kubernetes/cni/net.d \
```

Comment:
Why are you adding this instead of --cni-conf-dir, as per the official CoreOS docs?